### PR TITLE
Add a package .props file template

### PR DESCRIPTION
**Pull request by jtkech**
Review the change i made in the powerShell script

Related to : https://github.com/OrchardCMS/OrchardCore/pull/4346

### CODE REVIEW

**Comment by hishamco** [comment]
What about the meta package?!!

File: pack.ps1 [hunk 1]
```diff
@@ -25,6 +27,15 @@ function createNuGetPackage([string]$pkgName, [string]$culture)
     $cultureFolder = [IO.Path]::Combine($localizationFolderName, $culture)
     Copy-Item -Path $cultureFolder -Destination $pkgCultureFolderPath -Recurse
     
+    echo "Copying '.props' template file .."
```
Comment (hishamco) [inline pack.ps1]: echo "Copying '$pkgName.$pkgPropsExtension' ..."

File: pack.ps1 [hunk 2]
```diff
@@ -25,6 +27,15 @@ function createNuGetPackage([string]$pkgName, [string]$culture)
...
+
+    $pkgBuildFolderPath = [IO.Path]::Combine($pkgFolderPath, "buildTransitive")
```
Comment (hishamco) [inline pack.ps1]: Can we introduce a variable that hold "buildTransitive" like what we did for all the folders

**Comment by jtkech** [comment]
Normally nothing to do, this is each individual package that will enlist their po files in the msbuild var.

Can you review my powershell update?

File: pack.ps1 (hunk 1 above)
Comment (jtkech) [reply pack.ps1]: Okay

File: pack.ps1 (hunk 2 above)
Comment (jtkech) [reply pack.ps1]: Okay, i did it like that because you did the same before for `content`

File: pack.ps1 (hunk 1 above)
Comment (jtkech) [reply pack.ps1]: done

**Comment by hishamco** [comment]
> Normally nothing to do, this is each individual package that will enlist their po files in the msbuild var.

You 're right

LGTM after the little modification that I suggest

thanks

File: pack.ps1 (hunk 2 above)
Comment (jtkech) [reply pack.ps1]: done

File: pack.ps1 (hunk 1 above)
Comment (hishamco) [reply pack.ps1]: It's fine, because it's used once .. just leave it as it is

File: pack.ps1 (hunk 2 above)
Comment (hishamco) [reply pack.ps1]: It's fine, because it's used once .. just leave it as it is

File: pack.ps1 [hunk 3]
```diff
@@ -27,9 +28,9 @@ function createNuGetPackage([string]$pkgName, [string]$culture)
     $cultureFolder = [IO.Path]::Combine($localizationFolderName, $culture)
     Copy-Item -Path $cultureFolder -Destination $pkgCultureFolderPath -Recurse
     
-    echo "Copying '.props' template file .."
+    echo "Copying '$pkgName.$pkgPropsExtension' ..."
 
-    $pkgBuildFolderPath = [IO.Path]::Combine($pkgFolderPath, "buildTransitive")
+    $pkgBuildFolderPath = [IO.Path]::Combine($pkgFolderPath, $pkgBuildFolderName)
```
Comment (hishamco) [inline pack.ps1]: Seems you did before you saw my comment, coz there's no need for this change while it appears once

It's my fault during review, sorry

Comment (jtkech) [reply pack.ps1]: no problem

**Pull request merged**